packages/manager/modules/web-paas/src/onboarding/onboarding.routing.js

```diff
@@ -0,0 +1,18 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('web-paas.onboarding', {
+    url: '/onboarding',
+    component: 'webPaasOnboarding',
+    // redirectTo: (transition) =>
+    //   transition
+    //     .injector()
+    //     .getAsync('projects')
+    //     .then((projects) =>
+    //       projects.length > 0
+    //         ? { state: 'web-paas' }
+    //         : false,
+    //     ),
```

Comment:
Maybe we should prevent the access to onboarding if there are service right ?

Reply:
Yes, this is a temporary thing to let everyone see the onboarding page. I will remove this before it goes to QA